server/models/contests/updateQuestion.js

```diff
@@ -0,0 +1,36 @@
+const { pool } = require('../database')
+
+/**
+ * @param {*} param0
+ * @param {Object} param0.body
+ * @param {Object} param0.params
+ * @param {String} param0.username
+ * @return {Promise}
+ */
+
+function updateQuestion({ params, body, username }) {
+  return new Promise((resolve, reject) => {
+    const { contest_id: contestId, question_id: questionId } = params
+    const { max_score: maxScore } = body
+    pool.query(
+      `UPDATE contests_questions SET max_score = ? WHERE question_id = ? 
+        AND contest_id = (SELECT contest_id FROM contests_moderators WHERE contest_id = ? AND moderator = ?)`,
```

Comment:
```suggestion
        AND contest_id = ? AND EXISTS (SELECT 1 FROM contests_moderators WHERE contest_id = ? AND moderator = ?)`,
```
EXISTS works faster, as it will stop when it even finds one row which matches the WHERE condition, but SELECT statement will try find all such rows.